TYPO3.TypoScript/Classes/TYPO3/TypoScript/Core/Cache/ContentCache.php

```diff
@@ -237,11 +243,14 @@ public function getCachedSegment($uncachedCommandCallback, $typoScriptPath, $cac
     public function replaceCachePlaceholders(&$content, $addCacheSegmentMarkersToPlaceholders)
     {
         $cache = $this->cache;
+        $debugger = $this->debugger;
```

Comment:
@100hz aber die function hat doch einen Pointer auf die Instanz, ist ja keine statische Funktion. Da kann man doch auf $this zugreifen, auch wenn die Funktion sonst wie übergeben wird, weil das in diesem Singleton drinnen ist.